src/herder/test/HerderTests.cpp

```diff
@@ -1348,21 +1348,28 @@ TEST_CASE("quick restart", "[herder][quickRestart]")
     auto validatorKey = SecretKey::fromSeed(sha256("validator"));
     auto listenerKey = SecretKey::fromSeed(sha256("listener"));
 
+    auto validatorCfg =
+        getTestConfig(1, Config::TestDbMode::TESTDB_ON_DISK_SQLITE);
+    auto listenerCfg =
+        getTestConfig(2, Config::TestDbMode::TESTDB_ON_DISK_SQLITE);
+
     SCPQuorumSet qSet;
     qSet.threshold = 1;
     qSet.validators.push_back(validatorKey.getPublicKey());
 
-    auto validator = simulation->addNode(validatorKey, qSet);
-    auto listener = simulation->addNode(listenerKey, qSet);
+    simulation->addNode(validatorKey, qSet, &validatorCfg);
```

Comment:
we should not change this test like you did: this test is for testing that we send the right messages when a node drops off the network for a while.

There is a different test right above: "SCP State" that should be modified. Right now it only close 1 ledger (as before this PR only 1 ledger was persisted), but now we should be able to close a few ledgers (like 5).